nms/babel.config.js

```diff
@@ -11,33 +11,46 @@
  * limitations under the License.
  */
 
-module.exports = {
-  ignore: ['node_modules'],
-  presets: [
-    [
-      '@babel/preset-env',
-      {
-        targets: {
-          node: 'current',
-          chrome: '58',
+function isWebpack(caller) {
+  return !!(caller && caller.name === 'web');
```

Comment:
Why `!!`? Would `( a && a.name === 'web')` not already give a truthy back?

Reply:
I just followed there example https://babeljs.io/docs/en/config-files#apicallercb :) 
It is not clear to me how exactly the return value is used internally to configure the cache, so it might make a difference if there are three or two possible return values.